src/main/java/persistence/sql/ddl/impl/ColumnOptionSupplier.java

```diff
@@ -0,0 +1,46 @@
+package persistence.sql.ddl.impl;
+
+import jakarta.persistence.Column;
+import persistence.sql.ddl.QueryColumnSupplier;
+import persistence.sql.ddl.node.FieldNode;
+
+/**
+ * 컬럼 옵션을 처리하는 컬럼 쿼리 제공자
+ */
+public record ColumnOptionSupplier(short priority) implements QueryColumnSupplier {
+
+    @Override
+    public boolean supported(FieldNode fieldNode) {
+        return fieldNode != null;
+    }
+
+    @Override
+    public String supply(FieldNode fieldNode) {
+        StringBuilder constraint = new StringBuilder();
+
+        if (isNotNull(fieldNode)) {
```

Comment:
```suggestion
        if (fieldNode.isNotNull()) {
```

혹시, 이런식의 패턴은 어떻게 생각하시나요? 🤔

Reply:
이 부분 고민을 했는데 그렇게되면 FieldNode에서 평가되야 할 부분이 너무 늘어나는 것 같다는 생각이 들더라구요. 그렇게되다보면 Supplier보단 FieldNode에 책임들이 과중해질꺼 같기도하고, node.isNotNull() 함수는 NOT NULL 조건을 표현하기가 애매하더라구요.

Reply:
오 좋습니다 👍